php/ThreatExchangeConfig.php

```diff
@@ -23,7 +23,7 @@ class ThreatExchangeConfig {
   const APP_ID = 0;
   const APP_SECRET = 'secret';
 
-  const FACEBOOK_SERVER = 'https://graph.facebook.com';
+  const FACEBOOK_SERVER = 'https://graph.facebook.com/v2.4';
```

Comment:
I attempted that originally, but got this error when trying to use descriptors:

{"error":{"message":"(#12) Unable to display the requested object. requires version v2.4 or higher","type":"OAuthException","code":12}}`